src/main/java/command/Command.java

```diff
@@ -0,0 +1,23 @@
+package command;
+
+import executor.CommandExecutor;
+
+public class Command {
+
+    String input;
+
+    CommandExecutor commandExecutor;
```

Comment:
Any reason why you use CommandExecutor instead of just an execute command? Command.execute() seems simpler than Command.getCommandExecutor().execute()?